internal/gitops/structure/templates/management-cluster/template.go

```diff
@@ -0,0 +1,15 @@
+package organization
+
+import (
+	_ "embed"
+)
+
+//go:embed management-cluster.yaml.tmpl
+var managementCluster string
+
+// GetManagementClusterTemplate merges .tmpl files for organization layer.
+func GetManagementClusterTemplate() []string {
```

Comment:
Just a minor thing, but I would call it `GetManagementClusterTemplates` (in plural) because it could potentially return a list of templates.